src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/FormContext.cs

```diff
@@ -29,6 +29,14 @@ public class FormContext
             }
         }
 
+        public bool HasFormData
```

Comment:
@pranavkm why? Nothing in our guidelines requires Pranav points and nothing else in this class uses newer C# features.